crates/ruff/src/args.rs

```diff
@@ -974,12 +974,13 @@ A `--config` flag must either be a path to a `.toml` configuration file
             .is_some_and(|ext| ext.eq_ignore_ascii_case("toml"))
         {
             if !value.contains('=') {
-                tip.push_str(&format!(
+                let _ = write!(
+                    &mut tip,
                     "
 
 It looks like you were trying to pass a path to a configuration file.
 The path `{value}` does not point to a configuration file"
-                ));
+                );
```

Comment:
if we expect this `write!()` call to always succeed, would `.unwrap()` or `.expect()` be better?

Reply:
I thought about that and it actually was the first thing I wrote. 

I went with the lint's recommended fix because I expect most users to do the same, and this approach will lead to more consistent code.